pkg/varnishcluster/controller/varnishcluster_grafana_dashboard.go

```diff
@@ -194,7 +194,9 @@ func (r *ReconcileVarnishCluster) createGrafanaDashboardConfigMap(instance *vcap
 func generateGrafanaDashboardData(instance *vcapi.VarnishCluster) (map[string]string, error) {
 	data := map[string]interface{}{
 		"DatasourceName": *instance.Spec.Monitoring.GrafanaDashboard.DatasourceName,
-		"Title": instance.Spec.Monitoring.GrafanaDashboard.Title,
+		"Title":          instance.Spec.Monitoring.GrafanaDashboard.Title,
+		"ServiceName":    instance.Name,
```

Comment:
Also why call it `ServiceName` when it's the name of the `VarnishCluster`?

Reply:
It references the `service` label of the metric. And the Kubernetes service that exposes Varnish pods is called by the name of the VarnishCluster. It doesn't have an additional suffix.